### PR TITLE
ncm-*: Trim whitespace from Pan templates

### PR DESCRIPTION
**Pull request by jrha**
* Convert all tabs to four spaces.
* Remove trailing whitespace.
* Collapse multiple spaces within type declarations.
* Add missing newlines to end of files.

Also simplifies one of the pieces of test data used by ncm-named to make it not depend on whitespace.

When merged, this will likely cause all other open PRs on this repository to have conflicts, so it should probably get merged immediately following the release of 16.12.

### CODE REVIEW

**Comment by jrha** [comment]
Updated again to fix conflicts.

**Comment by stdweird** [comment]
@jrha @ned21 i propose to merge this either before 16.12 (say after rc0) or right after 16.12 is released.
with the new maventools there will be a new round of template cleanup/refactoring, so this PR will only cause conflicts with future work

File: ncm-accounts/src/main/pan/components/accounts/sysgroups.pan [hunk 1]
```diff
@@ -93,9 +93,9 @@ unique template components/accounts/sysgroups;
     'webalizer', '',
     'cimsrvr', '',
     'vboxusers', '',
-    'apacheds', '',
-    'mongod' , '',
-    'mongodb' , '',
-    'rabbitmq' , '',
-    'memcached' , '',
-   );
+    'apacheds', '',
```
Comment (stdweird) [inline ncm-accounts/src/main/pan/components/accounts/sysgroups.pan]: @jrha looks like a space before the comma is inserted. is this a bug in the script or in the github diff viewer?

File: ncm-accounts/src/main/pan/components/accounts/sysgroups.pan [hunk 2]
```diff
@@ -93,9 +93,9 @@ unique template components/accounts/sysgroups;
...
     'cimsrvr', '',
     'vboxusers', '',
-    'apacheds', '',
-    'mongod' , '',
-    'mongodb' , '',
-    'rabbitmq' , '',
-    'memcached' , '',
-   );
+    'apacheds', '',
+    'mongod' , '',
```
Comment (stdweird) [inline ncm-accounts/src/main/pan/components/accounts/sysgroups.pan]: looks like a space before the comma is inserted. is this a bug in the script or in the github diff viewer?

File: ncm-accounts/src/main/pan/components/accounts/sysgroups.pan (hunk 1 above)
Comment (stdweird) [inline ncm-accounts/src/main/pan/components/accounts/sysgroups.pan]: remark is for line below.

File: ncm-accounts/src/main/pan/components/accounts/sysgroups.pan (hunk 2 above)
Comment (jrha) [inline ncm-accounts/src/main/pan/components/accounts/sysgroups.pan]: Hmpf, it's genuine, one sec.

File: ncm-accounts/src/main/pan/components/accounts/sysgroups.pan (hunk 1 above)
Comment (jrha) [inline ncm-accounts/src/main/pan/components/accounts/sysgroups.pan]: fixed

**Comment by jrha** [comment]
I've sorted out the commas now and rebased again.

**Comment by stdweird** [comment]
@jrha i merged assuming the fixes wrt whitespaces are part of upcoming panlint PR...
